recipes/rfmix/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: rfmix
+  version: '2.9505bfa'
```

Comment:
where did you got this version ? the rfmix help says:
`RFMIX v2.03-r0`

Reply:
There are no formal releases (only the repository), so I used the most recent git commit prefix as the specifier and linked the download to that specific commit so that it is stable. I could change it to 2.03-r0, but that's not a stable, set, tarball release.

Reply:
Maybe 2.03-r0.9505bfa ?

Reply:
oh I see, thats the commit prefix. yes I think its a good idea. 
how about: `2.03-r0-9505bfa`
and add a comment around the version code indicating the commit prefix was added.

Reply:
Sounds good. Will do.

Reply:
uh oh, I forgot about the linting, use dots instead.